src/rules/no-default-export.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    // ignore non-modules
+    if (context.parserOptions.sourceType !== 'module') {
+      return {}
+    }
+
+    var message = 'Prefer named exports.'
```

Comment:
this should be a `const`.

also, i wonder if this is an opportunity here to make the message a bit more explanatory?

Reply:
ping

Reply:
Will do. I'll just note that the code base is a haphazard mix of `var`, `let`, and `const`, and ES6 features are used very inconsistently.

> also, i wonder if this is an opportunity here to make the message a bit more explanatory?

For the case of `export {foo as default}`, sure. For the rest, I'm not sure there is more that needs to be done IMHO. There's not really any extra context needed.